x-pack/test/functional/services/ml/single_metric_viewer.ts

```diff
@@ -117,5 +117,89 @@ export function MachineLearningSingleMetricViewerProvider({ getService }: FtrPro
       await testSubjects.click('mlAnomalyResultsViewSelectorExplorer');
       await testSubjects.existOrFail('mlPageAnomalyExplorer');
     },
+
+    async openConfigForControl(entityFieldName: string) {
+      const isPopoverOpened = await testSubjects.exists(
+        `mlSingleMetricViewerEntitySelectionConfigPopover_${entityFieldName}`
+      );
+
+      if (isPopoverOpened) {
+        return;
+      }
+
+      await testSubjects.click(
+        `mlSingleMetricViewerEntitySelectionConfigButton_${entityFieldName}`
+      );
+      await testSubjects.existOrFail(
+        `mlSingleMetricViewerEntitySelectionConfigPopover_${entityFieldName}`
+      );
+    },
+
+    // TODO move to some common service for EUI components
+    async assertRadioGroupValue(testSubject: string, expectedValue: string) {
+      const sortByRadioGroup = await testSubjects.find(testSubject);
+      const input = await sortByRadioGroup.findByCssSelector(':checked');
+      const selectedSortBy = await input.getAttribute('id');
```

Comment:
I think `selectedSortBy` should be named a bit more generic, maybe something like `selectedOptionId`.

Reply:
good spot, first I wrote it for this use-case and then forgot to rename during refactoring

Reply:
Renamed in https://github.com/elastic/kibana/pull/81510/commits/ca7c2fc0382dd05b98395d6264db8f2a43240f21